examples/pageon-metric.js

```diff
@@ -0,0 +1,33 @@
+import { browser } from 'k6/x/browser/async';
+
+export const options = {
+  scenarios: {
+    ui: {
+      executor: 'shared-iterations',
+      options: {
+        browser: {
+            type: 'chromium',
+        },
+      },
+    },
+  },
+}
+
+export default async function() {
+  const page = await browser.newPage();
+
+  page.on('metric', (metric) => {
+    metric.Tag({
+      urls: [
+        {urlRegEx: /^https:\/\/test\.k6\.io\/\?q=[0-9a-z]+$/, tagName:'test'},
```

Comment:
This was simpler like:

```js
{url: /^https:\/\/test\.k6\.io\/\?q=[0-9a-z]+$/, name:'test'},
```

My suggestion was about the Go code, not the JavaScript API. I'm sorry I didn't make it clearer back then 😢 The updated Go code is fine, but the previous JS API was more streamlined. Is it possible to revert it to the old API using `json:...` tags while keeping the updated Go code intact? Skip this suggestion if you prefer the new JS API.

Reply:
Right, ok, done in a6f9e3758130b139c9a872b6748b6d9b675f6a4e